README.md

```diff
@@ -1,2 +1,51 @@
-# lomax
-Workbench Archiving Service
+# Lomax
+Lomax is the Workbench Archiving Service.  It will handle bulk movement of GCS
+data into and out of archives, with an overall goal of saving users' money by
+taking advantage of cheaper storage classes for archives.
+
+This service is named after the legendary music archiver
+[Alan Lomax](https://en.wikipedia.org/wiki/Alan_Lomax).
+
+# For Developers
+
+## Setup
+This codebase requires *[npm](https://www.npmjs.com/get-npm)* and *[Node.js](https://nodejs.org/en/download/releases/)*.
+Specifically, it wants Node.js version 8.15.0, or whatever minor/patch version
+Google documents at https://cloud.google.com/functions/docs/concepts/nodejs-8-runtime.
+`npm` is included in the `Node.js` install so typically you do not have to install
+it separately.
+
+If you already have a different version of Node on your system, think you might
+need different versions in the future, or you have a hard time finding
+the right version of Node to install, you might be interested in *[nvm](https://github.com/creationix/nvm)*.
+`nvm` is not required to work with Lomax, but it is highly recommended and
+very useful to install and manage multiple versions of Node.
```

Comment:
FWIW, I prefer https://github.com/nodenv/nodenv for reasons: https://github.com/nodenv/nodenv/wiki/Why-nodenv%3F

Reply:
thanks - I will have to check that out!